### PR TITLE
Contact Me Feature

### PR DESCRIPTION
**Pull request by mohametalmeari**
Adding contact me section. 
Adding form inside the contact me section.
Styling the contact me section.

### CODE REVIEW

**Review by LionRouge1 — COMMENTED** [review]
Hi @mohametalmeari, 

done

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

You have submitted a project, that is a pair-programming activity, but you're the only one who committed to this project, please work with your coding partner and submit a PR to which you both contribute.

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

File: index.html [hunk 1]
```diff
@@ -183,5 +183,25 @@ <h3>
...
             </section>
     </div>
+    <section id="contact">
+        <h2>Contact me</h2>
+        <p>
+            If you have an application you are intersted in developing, 
+            a feature you that you need built or a project that needs coding. 
+            I'd love to help with it!
+        </p>
+        <form action="https://formspree.io/f/{form_id}" method="post">
```
Comment (ghost) [inline index.html]: - [x] Currently, your form submission raises an error. Kindly create a form with [formspree service](https://formspree.io/html) and use the unique id generated in your `action` prop.
 
image

**Review by frank1738 — CHANGES_REQUESTED** [review]
Hi Team, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!




## Highlights

✅ Correct use of Github workflow
✅ No linter error


## Required Changes ♻️

- [x] I think it would be better to indicate that the `live demo` link is not available for now if you have not deployed the project yet on your readme file. This would make your readme file more professional

_Check the comments under the review._

## Optional suggestions

N/A
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 1 above)
Comment (mohametalmeari) [reply index.html]: Hi @frank1738, thanks for the review, I did the changes, could check again before I request a re-review please.

**Review by aamirkhan2478 — APPROVED** [review]
Hi @mohametalmeari , 

GIF

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
✅ You used Gitflow correctly
✅ No linter error
✅ Good commit messages
✅ Good descriptive PR

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**